firestore-bigquery-export/firestore-bigquery-change-tracker/src/__tests__/bigquery/checkUpdates.test.ts

```diff
@@ -10,7 +10,7 @@ import {
 
 process.env.PROJECT_ID = "extensions-testing";
 
-const bq: BigQuery = new BigQuery();
+const bq: BigQuery = new BigQuery({ projectId: "extensions-testing" });
```

Comment:
This should run off the application default credentials, otherwise we could run across multiple projects by mistake? 

```suggestion
const bq: BigQuery = new BigQuery();
```